build/targets/common.props

```diff
@@ -1,6 +1,7 @@
 <Project>
   <PropertyGroup>
-    <BuildDir>$(RepositoryRoot)\build</BuildDir>
-    <ArtifactsDir>$(BuildDir)\artifacts</ArtifactsDir>
+    <RepositoryRoot Condition="'$(RepositoryRoot)'==''">$(MSBuildThisFileDirectory)..\..\</RepositoryRoot>
```

Comment:
This location can easily be incorrect, especially in (nested) test directories. Why isn't this using `$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), "$(Repository).sln")` or something similar?